pkg/compiler/codegen.go

```diff
@@ -758,6 +758,14 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 		}
 		// Do not swap for builtin functions.
 		if !isBuiltin {
+			if typ, ok := c.typeOf(n.Fun).(*types.Signature); ok && typ.Variadic() {
+				// pack variadic args into an array
+				varSize := len(n.Args) - typ.Params().Len() + 1
+				c.emitReverse(varSize)
```

Comment:
`Notify` code didn't reverse items, I'm not sure if it's a feature or a bug.

Reply:
It reversed them as a regular arguments, below this `if`.